components/k8s/pod/component_output.go

```diff
@@ -117,6 +122,20 @@ func CreateGet(cfg Config) query.GetFunc {
 
 		pods, err := ListFromKubeletReadOnlyPort(ctx, cfg.Port)
 		if err != nil {
+			// e.g.,
+			// Get "http://localhost:10255/pods": dial tcp 127.0.0.1:10255: connect: connection refused
+			connectionErr := false
+			if strings.Contains(err.Error(), "connection refused") {
+				connectionErr = true
+			}
+			if cfg.IgnoreConnectionErrors && connectionErr {
+				log.Logger.Warnw("failed to list pods from kubelet read-only port but ignoring since ignore_connection_errors is true", "error", err)
+				return &Output{
+					ConnectionError:        err.Error(),
+					ConnectionErrorIgnored: true,
+					Message:                "failed to list pods from kubelet read-only port but ignoring (maybe readOnlyPort not set in kubelet config file)",
+				}, nil
+			}
 			return nil, err
```

Comment:
return output, err instead of nil, err here?
we are ignoring errors for now but it might be unexpected if we start handling errors in the future..